runtime/onert/core/include/backend/ITensor.h

```diff
@@ -86,7 +86,7 @@ void setShape(ITensor *tensor, const ir::Shape &new_shape);
  * @brief Get ir::Shape of tensor
  * @note  Higer dimension will be placed on front.
  */
-ir::Shape getShape(ITensor *tensor);
+ir::Shape getShape(const ITensor *tensor);
```

Comment:
Could you also change this function to const function to like `ir::Shape getShape(const ITensor *tensor) const`?

Reply:
AFAIK, the last `const` is only available with method, not a function. :-)